.map.ts

```diff
@@ -0,0 +1,4 @@
+/** Auto-generated */
+declare const map: Record<string, unknown>
```

Comment:
Do all these files have to be in the root dir? 

Am a big fan of a clean root dir.